src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateServiceError.swift

```diff
@@ -84,7 +88,11 @@ enum HealthCertificateServiceError {
 				case .HC_PREFIX_INVALID:
 					return "\(AppStrings.HealthCertificate.Error.vcInvalid) (VC_PREFIX_INVALID)"
 				case .AES_DECRYPTION_FAILED:
-					return "\(AppStrings.HealthCertificate.Error.vcInvalid) (AES_DECRYPTION_FAILED)"
+					return "\(AppStrings.HealthCertificate.Error.vcInvalid) (AES_DECRYPTION_FAILED)."
+				case .HC_BASE45_ENCODING_FAILED:
+					return "\(AppStrings.HealthCertificate.Error.vcInvalid) (HC_BASE45_ENCODING_FAILED)."
+				case .HC_ZLIB_COMPRESSION_FAILED:
+					return "\(AppStrings.HealthCertificate.Error.vcInvalid) (HC_ZLIB_COMPRESSION_FAILED)."
```

Comment:
These error strings end up with a ".". The others not. Intentional?

Reply:
removed